### PR TITLE
Update README in MCT

### PR DESCRIPTION
**Pull request by kawakami-masaki0**
## Pull Request Description:
- Update README in MCT
  - Add CI/CD results of IMX500 Converter and TPC versions

Note: CICD errors occured because MCTWrapper verification repair is not yet supported.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,28 @@ Currently, MCT is being tested on various Python, Pytorch and TensorFlow version
...
 
+
+<br>
+
+
+It is also being tested with IMX500 Converter and TPC versions:
+<details id="supported-converter-versions">
+  <summary>Supported Versions Table (IMX500 Converter and TPC)</summary>
+
+| PyTorch 2.6           | TPC 1.0                                                                                                                                                                                                                                                           | TPC 4.0                                                                                                                                                                                                                                                           | TPC 5.0                                                                                                                                                                                                                                                           |
```
Comment (yt0705) [inline README.md]: >PyTorch 2.6

To avoid misunderstanding, let's delete it.

　Indeed, the model was created using PyTorch 2.6.
　However, PyTorch 2.6 does not run on the SDSP converter.

File: README.md [hunk 2]
```diff
@@ -146,6 +146,28 @@ Currently, MCT is being tested on various Python, Pytorch and TensorFlow version
...
+
+| TensorFlow 2.15       | TPC 1.0                                                                                                                                                                                                                                                           | TPC 4.0                                                                                                                                                                                                                                                           | TPC 5.0                                                                                                                                                                                                                                                           |
```
Comment (yt0705) [inline README.md]: > TensorFlow 2.15

To avoid misunderstanding, let's delete it.

**Review by yt0705 — CHANGES_REQUESTED** [review]

**Review by yt0705 — APPROVED** [review]
